Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -1077,99 +1071,216 @@ private protected void UpdateMaterialAssetByDefault(MaterialData src, ModelImpor
-                {
-                    destMat.EnableKeyword(_METALLICGLOSSMAP);
+            }
+
+            destMat.DisableKeyword(keyword);
+
+            return false;
+        }
+
+        private static Dictionary<string, string[]> synonymMap = new Dictionary<string, string[]> {
+            { MeshSyncConstants._Color, new[] { MeshSyncConstants._BaseColor } },
+            { MeshSyncConstants._MainTex, new[] { MeshSyncConstants._BaseMap, MeshSyncConstants._BaseColorMap } },
+            { MeshSyncConstants._Glossiness, new[] { MeshSyncConstants._Smoothness } },
+            { MeshSyncConstants._BumpMap, new[] { MeshSyncConstants._NormalMap } },
+            { MeshSyncConstants._EmissionMap, new[] { MeshSyncConstants._EmissiveColorMap } },
+            { MeshSyncConstants._ParallaxMap, new[] { MeshSyncConstants._HeightMap } },
+            { MeshSyncConstants._BumpScale, new[] { MeshSyncConstants._NormalScale } }
+        };
+
+        private static MaterialPropertyData? FindMaterialPropertyData(List<MaterialPropertyData> materialProperties, string name) {
+            for (int i = 0; i < materialProperties.Count; i++) {
+                if (materialProperties[i].name == name) {
+                    return materialProperties[i];
                 }
-                else if (propName == _BumpMap)
-                {
-                    destMat.EnableKeyword(_NORMALMAP);
+            }
+
+            return null;
+        }
+
+        private void ApplyMaterialProperty(MaterialData src, Material destMat, List<TextureHolder> textureHolders, MaterialPropertyData prop, string propName, List<MaterialPropertyData> materialProperties) {
+           
+            // Shaders use different names to refer to the same maps, this map contains those synonyms so we can apply them to every shader easily:
+            if (synonymMap.TryGetValue(propName, out var synonyms)) {
+                foreach (var synonym in synonyms) {
+                    ApplyMaterialProperty(src, destMat, textureHolders, prop, synonym, materialProperties);
+                }
+            }
+            
+            MaterialPropertyData.Type propType = prop.type;
+            if (!destMat.HasProperty(propName))
+                return;
+            
+            // Enable alpha test if there is a color texture so alpha clipping works:
+            if (propName == MeshSyncConstants._BaseMap ||
+                propName == MeshSyncConstants._MainTex || 
+                propName == MeshSyncConstants._BaseColorMap) {
+                bool hasAlpha = HandleKeywords(destMat, textureHolders, prop, MeshSyncConstants._ALPHATEST_ON);
+                if (hasAlpha) {
+                    destMat.SetOverrideTag("RenderType", "TransparentCutout");
                 }
+#if AT_USE_HDRP
+                destMat.SetFloat(MeshSyncConstants._AlphaCutoffEnable, hasAlpha ? 1 : 0);
+                if (hasAlpha) {
+                    destMat.renderQueue = 2475;
```

Comment:
Please use a value relative to one of the [RenderQueue enum](https://docs.unity3d.com/ScriptReference/Rendering.RenderQueue.html) values

Reply:
This doesn't have an enum value.

Reply:
I know, but we should put it relative to the existing enum values so that it's clear what is required.
Example: `Transparent + 75`

Reply:
Fixed